### ISSUE TITLE
LNURL withdraw

### ISSUE THREAD

**Issue by basharov** [muun/falcon#19]
I am playing around with lnurl-withdraw and seems like some other wallet (BlueWallet) works but Muun doesn't work the way I expect it to.

I have two urls in my webapp:
- http://192.168.0.24/api/payments/lnurl to receive the invoice details
- http://192.168.0.24/api/payments/lnurl/success to pay the invoice

Computer and my iPhone are in the same network and BlueWallet works with exactly the same server configuration.

1. I have a QR code
2. Scan the QR code
3. I see that my server returns both responses quite quickly - one after another.
4. Muun keeps showing "Receiving from 192.168.0.24".
5. Then it shows "192.168.0.24 is taking too long".
6. I click "Go to home" button.

Success endpoint just returns this:

```json
{
  "status": "OK"
}
```

May be Muun expects server to return something else?

**Comment by basharov** [comment]
@champo You're right. Actually the functionality to pay invoices is not yet implemented, so I tried to pay manually and it worked. When the invoice is paid, the success message is shown and the balance changes.

I just noticed that different wallets implement it differently - some just send the requests, then say "Wait for your invoice to be paid", others like Muun keep showing loading state.

Thanks anyway!
